Sources/Extensions/CustomTypes/LoginSessionConfiguration+OneLogin.swift

```diff
@@ -1,15 +1,42 @@
+import AppIntegrity
 import Authentication
 import Foundation
 
+protocol LoginSessionConfigurationProvider {
```

Comment:
I'm not sure what value this type gives, I think we should just pass in the configuration into the `PersistentSessionManager`.

Reply:
See comment above regarding `persistentID` which comes from the session manager.